target/linux/ramips/dts/mt7621_mikrotik_rbm11g.dts

```diff
@@ -138,14 +138,24 @@
 	};
 };
 
-&ethernet {
+&gmac0 {
 	mtd-mac-address = <&hard_config 0x0010>;
 	mtd-mac-address-increment = <1>;
 };
 
+&switch0 {
+	ports {
+		/* FIXME: Is LAN port at switch port 0? */
+		port@0 {
+			status = "okay";
+			label = "lan";
+		};
```

Comment:
Reminder: This needs to be checked/fixed.

Reply:
@TobleMiner ping